React-native.md

```diff
@@ -0,0 +1,84 @@
+# React-Native 
+
+React Native is a JavaScript framework Allow you to writes real, natively rendering mobile applications for iOS and Android ,based on React.
+You can create code just once and use it to power both their iOS and Android apps. This translates to huge time and resource savings. ,React Native has the ability to render APIs in Objective-C (for iOS) or Java (for Android),your application will render using “bridge” to become real mobile UI components,and will look and feel like any other mobile application,your React Native apps can access platform features like the phone camera, or the user’s location.
+
+![Test Image 1](https://www.netguru.com/hs-fs/hubfs/image10-Jul-03-2020-11-20-54-82-AM.png?width=1600&name=image10-Jul-03-2020-11-20-54-82-AM.png)
+
+Now we can start building our first React Native App ,to do that you can follow the steps below:
+### 1.Install React Native :
+Assuming that you not familiar with mobile development,you will need Node 12 LTS to use npm to install the Expo CLI command line utility:
+```
+$ npm install -g expo-cli
+```
+Then run the following commands to create a new React Native project :
+```
+$ expo init myFirstNative
+$ cd myFirstNative
+$ code .
+```
+
+### 2.let's modify it
+Open App.js in your text editor and edit some lines:
+
+```js
+import React from 'react';
+import { StyleSheet, Text, View, Button, Alert } from 'react-native';
+
+export default function App() {
+
+return (
+    <View style={styles.container}>
+      <Text style={styles.title}>This Is My First React Native App</Text>
+      <View style={styles.buttonContainer}>
+      <Button
+      title="click me"
```

Comment:
Some of the indentation is off here. Indentation is a really big deal in coding - it shows you care about your code and spend lots of time looking after it 👩‍🌾 👨‍🌾 

If you want to cheat and get your indentation correct for no extra effort, [Prettier](https://prettier.io/) can help. Prettier works particularly well with [Visual Studio Code](https://code.visualstudio.com/) (which is my favourite editor). Let me know if you need a hand getting any of this set up.